resources/views/pages/home.blade.php

```diff
@@ -2,18 +2,57 @@
 
 @section('main_content')
 
-    @include('layouts.header', ['header' => 'Rogue', 'subtitle' => 'Please log in to continue.'])
+    @include('layouts.header', ['header' => 'Rogue', 'subtitle' => 'Please log in with your DoSomething.org admin account to continue.
+'])
 
     <div class="container -padded">
         <div class="wrapper">
-            <div class="container__block -narrow">
+            <div class="container__block">
```

Comment:
It might be nice to make this a `.container__block.-narrow` so it's a bit easier to read.